packages/ndla-ui/src/Navigation/NavigationBox.tsx

```diff
@@ -122,20 +87,21 @@ const StyledButtonContent = styled.span`
   align-items: center;
 `;
 
-const StyledButtonContentText = styled.span<additionalResourceProps>`
-  ${(props) => {
-    if (props.isAdditionalResource && props.isRestrictedResource) {
-      return `padding-left: ${spacing.medium};`;
-    }
-    if (props.isAdditionalResource || props.isRestrictedResource) {
-      return `padding-left: ${spacing.small};`;
-    }
-  }}
+const StyledButtonContentText = styled.span`
+  &[data-additional='true'][data-restricted='true'] {
```

Comment:
Bare så det er sagt: CSS-spesifiteten man får av å matche på data-attributter er veldig høy. Dersom vi ser for oss å style denne komponenten med klassenavn et annet sted kommer det til å være vanskelig å overstyre eksisterende styling.

Reply:
Det er mulig, men eg har berre forsøkt å unngå props. Du må gjerne skrive dette om i etterkant.